tt_eager/tt_lib/csrc/tt_lib_bindings_tensor.cpp

```diff
@@ -100,6 +101,10 @@ void TensorModule(py::module &m_tensor) {
 
     detail::export_enum<ShardOrientation>(m_tensor);
 
+    detail::export_enum<LossFunction>(m_tensor);
+
```

Comment:
Any reason to export this to the user if we aren't exposing the function that uses this?